LKExp-Firmware_update-Serial_update/BOOTLOADER/lib/mbed-os.lib

```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/mbed-os/
```

Comment:
pareil on peut supprimer celui ci dans BOOTLOADER et le même dans APPLICATION

Reply:
Fix in commit f0bb12e
Il est présent dans le _template, il faudrait l'en retirer